sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerConfig.java

```diff
@@ -221,6 +226,10 @@ public String getDcosSpace() {
         return envStore.getOptional(MARATHON_APP_ID_ENV, "/");
     }
 
+    public String getSchedulerRegion() {
+        return envStore.getOptional(SERVICE_REGION_ENV, null);
```

Comment:
This follows the pattern in the rest of this file, but I wonder why we're not using `Optional` here and elsewhere.  Generally speaking we don't pass around `null` a lot in the SDK.

Reply:
That's sort of the confusing aspect of the `getOptional` pattern -- since it doesn't return an `Optional`, one has to indicate absence of value with null. I'd be happy to create a ticket to refactor that elsewhere.